src/elements/content-sidebar/versions/__tests__/VersionsSidebarContainer-test.js

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { shallow } from 'enzyme/build';
+import VersionsSidebar from '..';
+
+jest.mock('../../../common/api-context', () => ({
```

Comment:
I think this is preferable to exporting/importing unwrapped versions of components. It allows us to avoid modifying source code just to support unit tests. I'm am open to discussion about it, though.